src/keyword.rs

```diff
@@ -164,8 +164,9 @@ pub fn index(req: &mut Request) -> CargoResult<Response> {
 /// Handles the `GET /keywords/:keyword_id` route.
 pub fn show(req: &mut Request) -> CargoResult<Response> {
     let name = &req.params()["keyword_id"];
+    let lower_name = name.to_lowercase();
     let conn = try!(req.tx());
-    let kw = try!(Keyword::find_by_keyword(&*conn, &name));
+    let kw = try!(Keyword::find_by_keyword(&*conn, &lower_name[..]));
```

Comment:
The database already turns this into lowercase, so I don't think we need the explicit `to_lowercase` here